apps/studio/components/layouts/ProjectSettingsLayout/SettingsMenu.utils.tsx

```diff
@@ -156,13 +154,15 @@ export const generateSettingsMenu = (
           key: 'subscription',
           url: `/org/${organization?.slug}/billing`,
           items: [],
+          rightIcon: <ArrowUpRight strokeWidth={1} className="h-4 w-4" />,
         },
 
         {
           name: 'Usage',
           key: 'usage',
           url: `/org/${organization?.slug}/usage?projectRef=${ref}`,
           items: [],
+          rightIcon: <ArrowUpRight strokeWidth={1} className="h-4 w-4" />,
```

Comment:
These are technically shortcuts to other areas, so I’m making these visually consistent with the others via `ArrowUpRight` icons.